src/scripts/stats.py

```diff
@@ -150,20 +176,26 @@ async def main(config_key):
     df_agg['trades'] = df_agg['trades'].astype(int)
     df_agg['cumulativeTrades'] = df_agg['trades'].cumsum()
 
-    # Trader data query and processing
-    trader_query = config['queries']['traders']
-    df_trader = pd.DataFrame(await run_recursive_query(trader_query['query'], {'last_id': ''}, trader_query['accessor'], config['subgraph_endpoint'])).drop('id', axis=1).sort_values('timestamp')
-    df_trader['dateTs'] = df_trader['timestamp'].apply(lambda x: int(int(x) / 86400) * 86400)
-    df_trader['cumulativeTraders'] = (~df_trader['accountId' if config_key == 'v3' else 'account'].duplicated()).cumsum()
-    df_trader_agg = df_trader.groupby('dateTs')['accountId' if config_key == 'v3' else 'account'].nunique().reset_index()
-    df_trader_agg.columns = ['timestamp', 'uniqueTraders']
-    df_trader_agg['cumulativeTraders'] = df_trader.groupby('dateTs')['cumulativeTraders'].max().reset_index()['cumulativeTraders']
+    if config_key == 'perennial':
+        df_agg['volume'] = df_agg['longNotional'].astype(float) / 1000000 + df_agg['shortNotional'].astype(float) / 1000000
```

Comment:
We might use `_` as a thousands separator in `1_000_000` for better readability.